SEED-platform/seed
# Patch/bsync import

#### Any background context you want to provide?
I broke some things...
#### What's this PR do?
Fixes the following:
1. importing tax-lots would result in the user getting continually redirected to the data mapping page. This should no longer happen
2. forces all bsync files in uploaded zip to have valid schemaLocations before moving to mapping page
#### How should this be manually tested?
1. upload example PM spreadsheet, make all fields tax lot. You should be able to import normally
2. upload example PM spreadsheet, make all fields property. Should import normally
3. upload ex_1_no_schemaLocation_and_ex_1.zip and it should error immediately and not take you to the mapping page
4. upload a normal bsync file and it should work normally
5. upload ex_1_and_buildingsync_ex01_measures.zip and it should upload both properties normally
